src/types/index.ts

```diff
@@ -1 +1,42 @@
-export * from "./types";
+export enum StatusOptions {
+  WAIT = 1,
+  PENDING_UPDATE = 2,
+  COMPLETED = 3,
+  CANCELED = 4,
+  IN_PROGRESS = 5,
+  ALL_STATUSES = 6,
+}
+
+export type Statuses = {
+  [key: number]: {
+    hebrew_name: string;
+    color: string;
+  };
+};
+
+export type Task = {
+  taskId: number;
+  dueDate: string;
```

Comment:
why not Date type ?

Reply:
The field is an input of type 'date,' which requires a string. Since the server already returns a string, I want to avoid converting it twice.